### PR TITLE
[UI] ADH-4629: Add Cluster nodes table

### PR DESCRIPTION
**Pull request by alatunov**
https://tracker.yandex.ru/ADH-4629

### CODE REVIEW

File: smart-frontend/app/src/api/adh/clusterInfo.ts [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+ */
+import type { PaginateCollection } from '@models/collection';
+import { httpClient } from '@api/httpClient';
+import type { AdhClusterNodesFilter, AdhClusterNode } from '@models/adh';
+import type { PaginationParams, SortParams } from '@models/table';
+import { prepareDateRange, prepareQueryParams } from '@utils/requestUtils';
+import qs from 'qs';
+
+export class AdhClusterInfoApi {
+  public static async getNodes(
```
Comment (remizov-arena) [inline smart-frontend/app/src/api/adh/clusterInfo.ts:26]: Why dose not use AdhClustersApi? Upgrade method and use it

File: smart-frontend/app/src/api/adh/clusterInfo.ts [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    { registrationTime, ...filter }: AdhClusterNodesFilter,
+    sortParams?: SortParams,
+    paginationParams?: PaginationParams,
+  ): Promise<PaginateCollection<AdhClusterNode>> {
+    const { from: registrationTimeFrom, to: registrationTimeTo } = prepareDateRange(registrationTime);
+
+    const queryParams = prepareQueryParams(
+      {
+        ...filter,
+        registrationTime: { registrationTimeFrom, registrationTimeTo },
```
Comment (remizov-arena) [inline smart-frontend/app/src/api/adh/clusterInfo.ts:36]: there is case when filter can be empty ( when  wegit hosts list for filter on actions page. Should catch it, and not send  registrationTime in this case

File: smart-frontend/app/src/store/adh/cluster/clusterSlice.ts [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+});
+
+const createInitialState = (): AdhClustersSliceState => ({
+  nodes: [],
+  totalCount: 0,
+  loadState: LoadState.NotLoaded,
+  liveCount: 0,
+});
+
+const clustersSlice = createSlice({
```
Comment (remizov-arena) [inline smart-frontend/app/src/store/adh/cluster/clusterSlice.ts]: * try that file name will be equal slice name
* this slice not about cluster, this slice about cluster nodes. I think will be better named as `clusterNodesSlice` - few nodes of one cluster

Comment (alatunov) [reply smart-frontend/app/src/store/adh/cluster/clusterSlice.ts]: Slices names are changed

File: smart-frontend/app/src/api/adh/clusterInfo.ts (hunk 2 above)
Comment (alatunov) [reply smart-frontend/app/src/api/adh/clusterInfo.ts:36]: if `registrationTimeFrom, registrationTimeTo` are undefined then `registrationTime` will not be sent in query

**Review by remizov-arena — APPROVED** [review]

**Pull request merged**